Hosting API Documentation.md

```diff
@@ -0,0 +1,112 @@
+# Summary
+
+This is the flow we are using to generate and host our documentation:
+
+- Write tests using rspec and annotate them using [dox](https://github.com/infinum/dox)
+- Once tests on [semaphore](https://semaphoreci.com) pass we move to deploy
+- In the deploy phase we do the following:
+  * deploy the application using [mina](https://github.com/mina-deploy/mina)
+  * run only dox tests to generate the documentation
+  * copy the generated documentation to the application server inside `public` folder
+
+# Why
+
+We settled on serving our documentation for the application server because it is convenient, easiest
```

Comment:
Maybe something like:

Our API documentation is exposed on the application server, which makes it convenient, easy to use and free.